### ISSUE TITLE
Accessibility: Firefox inspector audit

### ISSUE THREAD

**Issue by BeckyPollard** [BeckyPollard/personal-site-v3#1]
## TODO
In `index.html`

- Images (svgs) do not have any label, which violates accessibility. The SVGs have a <title> within them, but they still flag errors in the Firefox accessibility check.
- Text contrast is too low in contact disclaimer.
- Text contrast is too low in hidden black-on-black text in the About section **(will not fix)**.
